neqo-crypto/src/aead_fuzzing.rs

```diff
@@ -0,0 +1,68 @@
+// Licensed under the Apache License, Version 2.0 <LICENSE-APACHE or
+// http://www.apache.org/licenses/LICENSE-2.0> or the MIT license
+// <LICENSE-MIT or http://opensource.org/licenses/MIT>, at your
+// option. This file may not be copied, modified, or distributed
+// except according to those terms.
+
+use crate::constants::{Cipher, Version};
+use crate::err::{Error, Res};
+use crate::p11::SymKey;
+use std::fmt;
+
+pub struct Aead {}
+
+impl Aead {
+    pub fn new(_version: Version, _cipher: Cipher, _secret: &SymKey, _prefix: &str) -> Res<Self> {
+        Ok(Self {})
```

Comment:
Can you add something that causes this code to explode if it is run outside of a debug build?

```rust
#[cfg(not(debug_assertions))]
panic!("Trying to run a non-debug fuzzing build.");
```